### ISSUE TITLE
Cannot scroll chat window without scroll wheel?

### ISSUE THREAD

**Issue by RollingStar** [Hergan5/herganhud#1]
Does scrolling the chat window require a scroll wheel? If so, is there any way to change this?

**Comment by Hergan5** [comment]
I don't think so, the chat menu has arrows on the right of it, you can use that, not sure if you can make the scrolling on other key or something

**Comment by RollingStar** [comment]
![image](https://cloud.githubusercontent.com/assets/4810902/7801128/f73b0b36-02ed-11e5-9ba4-727f592cf3bd.png)

Maybe I installed it wrong? I usually run in borderless windowed 1600x900. (For some reason Steam seems to have lost all of my launch options for all of my Steam games.)

**Comment by Hergan5** [comment]
Huh, didn't even noticed that, can you try deleting ChatScheme from resource and BaseChat from resource/ui and tell me if it'll work?
